packages/rancher-sriov/generated-changes/exclude/crds/k8s.cni.cncf.io_networkattachmentdefinitions_crd.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 NVIDIA
```

Comment:
I see `network-attachment-definition` is listed in the clusterrole `sriov-network-operator`. 
It is ok to include it in the clusterrole even the CRD does not exist. I just need to confirm that we do want to exclude this CRD.